### PR TITLE
Improvement in generate_load.py

### PR DESCRIPTION
**Pull request by realsystem**
Added support for cleaning security-groups.
New argument can be used for choosing cloud.

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by plomakin** [comment]
+1 looks good.

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

File: devlab/tests/generate_load.py [hunk 1]
```diff
@@ -17,7 +17,6 @@
 NEUTRON_CLIENT_VERSION = config.NEUTRON_CLIENT_VERSION
 CINDER_CLIENT_VERSION = config.CINDER_CLIENT_VERSION
 
-
```
Comment (roman-verchikov) [inline devlab/tests/generate_load.py]: pep8 violation (minor)

File: devlab/tests/generate_load.py [hunk 2]
```diff
@@ -431,15 +443,15 @@ def clean_objects(self):
                 print "Volume %s failed to delete: %s" % (volume['name'],
                                                           repr(e))
 
-
```
Comment (roman-verchikov) [inline devlab/tests/generate_load.py]: pep8 violation (minor)

**Comment by roman-verchikov** [comment]
Looks good other than 2 pep8 whitespace violations identified inline. Please fix

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by roman-verchikov** [comment]
+1, can be merged.

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Pull request merged**